app/governance/components/ProposalTable/ProposalTable.tsx

```diff
@@ -0,0 +1,310 @@
+
+  if (!proposals || proposals.length == 0) {
+    return (
+      <div>
+        <Text font="proto_mono">Loading Proposals...</Text>
+      </div>
+    );
+  }
+  return (
+    <div className={styles.tableContainer}>
+      <div className={styles.table}>
+        {
+          <Table
+            title={proposalTitleMap.get(currentFilter)}
+            secondary={
+              <Container width="400px">
+                <ToggleGroup
+                  options={["All", "Active", "Passed", "Rejected"]}
+                  selected={currentFilter}
+                  setSelected={(value) => {
+                    setCurrentFilter(value);
+                  }}
+                />
+              </Container>
+            }
+            headers={
+              filteredProposals.length != 0 || filteredProposals
+                ? [
+                    {
+                      value: (
+                        <Text opacity={0.4} font="rm_mono">
+                          ID
+                        </Text>
+                      ),
+                      ratio: 2,
+                    },
+                    { value: <Text opacity={0.4}>Title</Text>, ratio: 6 },
+                    {
+                      value: (
+                        <Text opacity={0.4} font="rm_mono">
+                          Status
+                        </Text>
+                      ),
+                      ratio: 3,
+                    },
+                    {
+                      value: (
+                        <Text opacity={0.4} font="rm_mono">
+                          Type
+                        </Text>
+                      ),
+                      ratio: 5,
+                    },
+                    {
+                      value: (
+                        <Text opacity={0.4} font="rm_mono">
+                          Voting Date
+                        </Text>
+                      ),
+                      ratio: 4,
+                    },
+                  ]
+                : []
+            }
+            content={
+              paginatedProposals.length > 0
+                ? [
+                    ...paginatedProposals.map((proposal, index) => {
+                      return (
+                        <div
+                          key={`row_${index}`}
+                          style={{
+                            display: "flex",
+                            flexDirection: "row",
+                            cursor: "pointer",
+                            alignItems: "center",
+                            justifyContent: "center",
+                            height: "80px",
+                          }}
+                        >
+                          <Container
+                            width="10%"
+                            key={`name_${index}`}
+                            style={{ cursor: "pointer", alignItems: "center" }}
+                            onClick={() => handleRowClick(proposal.proposal_id)}
```

Comment:
looks like you are passing in duplicate onClicks for each item the the row. you can just add onClick to the row itself

Reply:
Done. Changed the code